third_party/packages/perfetto_compiled/lib/dist/index.html

```diff
@@ -8,7 +8,7 @@
   <link id="devtools-style" rel="stylesheet" href="devtools/devtools_dark.css">
   <script src="devtools/devtools_theme_handler.js"></script>
 </head>
-<body data-perfetto_version='{"stable":"v31.0-61eeba8d0"}'>
```

Comment:
all changes from this file down are caused by refreshing the perfetto assets to the latest. These do not need to be reviewed.